src/views/tickets/rsvp.php

```diff
@@ -168,7 +168,7 @@ class="tribe-ticket-quantity"
 
 						<tr class="tribe-tickets-order_status-row">
 							<td>
-								<label for="tribe-tickets-order_status"><?php echo esc_html_x( 'RSVP', 'order status label', 'event-tickets' ); ?>:</label>
+								<label for="tribe-tickets-order_status"><?php echo tribe_get_rsvp_label_singular( 'order_status_label' ); ?>:</label>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 2fa1083f7c92df60e54c13ee69a05a0f</sub>